.github/workflows/translations.yaml

```diff
@@ -3,12 +3,10 @@ name: Translations
 on:
   push:
     branches:
-      - main
-      - 'releases/**'
+      - 'releases/2.15.*'
```

Comment:
Why remove `main` here (and below)? Could we keep it all - make this run on `releases/**`, `releases/2.15.*`, and `main`?

Reply:
oops sorry, just seeing this comment otherwise I wouldn't have force pushed but this was find/replace pasta and is now reverted